docs/src/tutorial/clibraries.rst

```diff
@@ -256,7 +260,7 @@ if other applications also use C-Alg.
 Static Linking
 ---------------
 
-To build the c-code automatically we need to include compiler directives in `queue.pyx`
```

Comment:
we can also use
```
:file:`<name>`
```
for all of those filenames

and it will be better, i think

Reply:
I was not aware about that syntax. Thank you!